src/components/Menu/MenuDivider/__stories__/MenuDivider.stories.js

```diff
@@ -0,0 +1,57 @@
+import MenuDivider from "../MenuDivider";
+import { createComponentTemplate } from "vibe-storybook-components";
+import Menu from "../../Menu/Menu";
+import MenuItem from "../../MenuItem/MenuItem";
+
+export default {
+  title: "Navigation/Menu/MenuDivider",
+  component: MenuDivider
+};
+
+const menuDividerTemplate = createComponentTemplate(MenuDivider);
```

Comment:
Why not just write the code? We don't bind it anywhere

Reply:
We bind it in overview example and provide args for it